lib/msfdb_helpers/pg_ctl.rb

```diff
@@ -20,6 +21,17 @@ def init(msf_pass, msftest_pass)
         f.puts "port = #{@options[:db_port]}"
       end
 
+      # Try creating a test file at {Dir.tmpdir},
+      # Else fallback to creation at @{db}
+      # Else fail with error.
+      if test_executable_file("#{Dir.tmpdir}")
+        @socket_directory = Dir.tmpdir
+      elsif test_executable_file("#{@db}")
+        @socket_directory = @db
+      else
+        print_error("Attempt to create DB socket file at Temporary Directory and `~/.msf4/db` failed. Possibly because they are mounted with NOEXEC Flags. Database initialization failed.")
```

Comment:
```suggestion
        print_error("Attempt to create DB socket file at Temporary Directory and `~/.msf4/db` failed. Possibly because they are mounted with NOEXEC flags. Database initialization failed.")
```

Reply:
Fixed with 19d044621c79be99f16427005f0eafac1337da10